geoportailv3/templates/index.html

```diff
@@ -145,7 +145,9 @@ <h2 translate>my_maps</h2>
             ✕
           </button>
           <div class="col-xs-12">
-            <app-mymaps><app-mymaps>
+            <app-mymaps app-mymaps-selectinteraction="mainCtrl.selectInteraction"
+              app-mymaps-features="mainCtrl.drawnFeatures">
```

Comment:
You should use one-time binding for both `selectInteraction` and `drawnFeatures`.